eos-core-depends

```diff
@@ -58,3 +58,94 @@ unoconv
 unrar
 whiptail
 xdg-app
+
+
+# moved here from os-depends due to installer
+chromium-browser
+cups
+eos-social
+eos-speedwagon
+eos-tech-support
+eos-tutorial
+eos-updater
+evince
+fake-hwclock
```

Comment:
I would keep this as it handles the persistent system clock on system's without a hardware clock.